src/user/faq.rst

```diff
@@ -71,4 +71,10 @@ FAQ
 
     x86_64-apple-darwin13.4.0-clang: No such file or directory
     
-  are a telltale sign that you are lacking compilers.
\ No newline at end of file
+  are a telltale sign that you are lacking compilers.
+
+:ref:`(Q) <faq_cuda_compiler_header>` **How can I compile CUDA (host or device) codes in my environment?**
+
+  Unfortunately, this is not possible with Conda-Forge's current infrastructure (``nvcc``, ``cudatoolkit``, etc) if there is no local CUDA Toolkit installation. In particular, the ``nvcc`` package provided on Conda-Forge is a *metapackage* wrapping an actual ``nvcc`` compiler for our CI to use; it does not contain the full ``nvcc`` compiler toolchain. One of the reasons is that CUDA headers like ``cuda.h``, ``cuda_runtime.h``, etc, are not redistributable according to NVIDIA's EULA, which are needed at compile time. Likewise, the ``cudatoolkit`` package only contains CUDA runtime libraries and not the compiler toolchain.
```

Comment:
Nitpicking here, feel free to ignore:

In the maintainer bits I referred to `nvcc` as a *wrapper package* since it doesn't provide the compilers directly, just wraps the system install with some activation scripts. IMO a *metapackage* is a package that only lists other packages as dependencies, but doesn't deal with anything else like complex mechanisms to set environment variables.

Thus, I'd say something like this:


```suggestion
  Unfortunately, this is not possible with Conda-Forge's current infrastructure (``nvcc``, ``cudatoolkit``, etc) if there is no local CUDA Toolkit installation. In particular, the ``nvcc`` package provided on Conda-Forge is a *wrapper package* that exposes the actual ``nvcc`` compiler to our CI infrastructure in `conda`-friendly way; it does not contain the full ``nvcc`` compiler toolchain. One of the reasons is that CUDA headers like ``cuda.h``, ``cuda_runtime.h``, etc, are not redistributable according to NVIDIA's EULA, which are needed at compile time. Likewise, the ``cudatoolkit`` package only contains CUDA runtime libraries and not the compiler toolchain.
```

Reply:
Sounds good, thanks Jaime!

Reply:
Awesome, thanks for the work here!

Reply:
Oops, and sorry about the single backtick thing. Too much markdown 😬

Reply:
No worries. I don't know the syntax here either; simply following what's done above 😂